vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -205,20 +214,17 @@ export default class SbcHeader extends Mixins(NavigationMixin) {
 
   get showAccountSwitching (): boolean {
     try {
-      const flags = JSON.parse(ConfigHelper.getFromSession(SessionStorageKeys.LaunchDarklyFlags) || '{}')
-      return flags && flags['account-switching']
+      return LaunchDarklyService.getFlag('account-switching')
```

Comment:
Why is this code (inside try..catch) different from code below (line 224), when they do the same thing?
If you're looking for a suggestion which way to go, I think the code below looks cleaner (if in fact it works as intended).

Reply:
It was already inside the try catch, but yeah, the below code does the same and works,
I will change this to make it consistant
Thanks @severinbeauvais